app/views/layouts/application.html.slim

```diff
@@ -28,4 +28,14 @@ html
     == render 'layouts/icons'
 
     javascript:
-      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
\ No newline at end of file
+      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
+
+    - if Rails.env.production?
```

Comment:
right now we're setting rails_env as production on dahlia-lap-full. we'd probably have to update that in order to not have dev analytics data coming through, right?

Reply:
That is correct, but we can set this with another variable if needed/wanted

Reply:
I guess another option is to just not set the tracking id, right? What happens if we don't set it?

Reply:
Yea we could just change the check to be for that tracking ENV VAR instead of overall environment, then if its not set non of that logic will be added, would you like me to go ahead with that?

Reply:
yeah let's maybe do check for production + google analytics tracking id? what do you think?

Reply:
OK, talked to Shawn for our team and he thinks the plan of going with just checking for the presence of the env var is OK. thanks!

Reply:
Updated